google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinInstruments.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub.metrics;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.api.metrics.Meter;
+
+class BuiltinInstruments extends BigtableInstruments {
+
+  private static final String MILLISECOND = "ms";
+  private static final String COUNT = "1";
+
+  private final LongHistogram operationLatencies;
+  private final LongHistogram attemptLatencies;
+  private final LongHistogram serverLatencies;
+  private final LongHistogram firstResponseLatencies;
+  private final LongHistogram clientBlockingLatencies;
+  private final LongHistogram applicationBlockingLatencies;
+  private final LongCounter connectivityErrorCount;
+  private final LongCounter retryCount;
+
+  BuiltinInstruments(Meter meter) {
+    operationLatencies =
+        meter
+            .histogramBuilder("operation_latencies")
```

Comment:
I see the name is different from the existing OpenCensus name `cloud.google.com/java/bigtable/operation_latencies`, because slashes are not allowed anymore? I know it will be overridden by the view names later so it probably does not matter for built in metrics, it will be a problem for non-built in metrics though.